src/test/java/LoginTests.java

```diff
@@ -14,12 +14,35 @@
 
 public class LoginTests extends BaseTest {
 
+    @Test
+    public void LoginValidEmailPasswordTest () {
+        LoginPage loginPage = new LoginPage(driver);
+        HomePage homePage = new HomePage(driver);
+
+        loginPage.provideEmail("demo@class.com");
+        loginPage.providePassword("te$t$tudent");
+        loginPage.clickSubmitBtn();
+        Assert.assertTrue(homePage.isUserAvatarDisplayed());
+
+    }
+
+    @Test
+    public void LoginEmptyPasswordTest () {
+        LoginPage loginPage = new LoginPage(driver);
+
+        loginPage.provideEmail("demo@class.com");
+        loginPage.providePassword("");
+        loginPage.clickSubmitBtn();
+        Assert.assertTrue(loginPage.isPageOpened());
+
+    }
+
```

Comment:
I fixed it:

![image](https://user-images.githubusercontent.com/104761482/201676789-4e03c6d7-dc85-442a-803e-fdd3016395c6.png)

![image](https://user-images.githubusercontent.com/104761482/201676865-e3fdcbea-bc3c-4c0a-83b0-8176d2dbcb2f.png)

Here is the solution:

![image](https://user-images.githubusercontent.com/104761482/201676953-c6c7b451-3d50-40d1-b947-b938d76b994f.png)

Reply:
I've tried all locators and sometimes it works and sometimes it doesn't.